metagpt/document_store/base_store.py

```diff
@@ -15,7 +15,7 @@ class BaseStore(ABC):
     """FIXME: consider add_index, set_index and think 颗粒度"""
 
     @abstractmethod
-    def search(self, query, *args, **kwargs):
+    def search(self, *args, **kwargs):
```

Comment:
Because *args and **kwargs already can include all params, if abstract Base Class function query is first param, sub class also must same, I think this is unreasonable, and pycharm will warning